.github/workflows/terraform-plan-apply.yml

```diff
@@ -220,3 +220,12 @@ jobs:
           # first clean up the container, then upload the new assets
           az storage blob delete-batch --connection-string ${ui_storage_conn_string} --source "\$web" --pattern "*"
           az storage blob upload-batch --connection-string ${ui_storage_conn_string} --source mts-trial-ui --destination '$web'
+
+  buildall:
```

Comment:
probably this should be "tfplanall" ?
but maybe this is a good time to rename to note this is actually deploying things? maybe simple deploy / deployall?